com.unity.cinemachine/Samples~/3D Samples/Trigger Volume.unity

```diff
@@ -0,0 +1,1298 @@
+%YAML 1.1
```

Comment:
I think we can come up with a better concept to show trigger volumes.  This just feels broken.  Character is hard to control, and cameras make no sense.  Original upon which this was based was pretty bad too.